bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -139,6 +143,19 @@ WebApp.connectHandlers.use('/feedback', (req, res) => {
   }));
 });
 
+WebApp.connectHandlers.use('/useragent', (req, res) => {
+  const userAgent = req.headers['user-agent'];
+  let response = 'No user agent found in header';
+  if (userAgent) {
+    response = lookupUserAgent(req.headers['user-agent']).toString();
```

Comment:
```suggestion
    response = lookupUserAgent(userAgent).toString();
```

Reply:
Good catch. I meant for it to be that and missed a spot.